flink-python/pyflink/table/table_environment.py

```diff
@@ -79,6 +81,8 @@ def __init__(self, j_tenv, serializer=PickleSerializer()):
         self._j_tenv = j_tenv
         self._is_blink_planner = TableEnvironment._judge_blink_planner(j_tenv)
         self._serializer = serializer
+        # for local executor, use current sys.executable to launch UDF worker.
```

Comment:
Update the comments as following:
`when running in MiniCluster, launch the Python UDF worker using the Python executable specified by sys.executable if users have not specified it explicitly via configuration python.executable.`